cypress/integration/transforms_spec.js

```diff
@@ -0,0 +1,238 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+import { PLUGIN_NAME } from "../support/constants";
+import sampleTransform from "../fixtures/sample_transform";
+
+const TRANSFORM_ID = "test_transform_id";
+
+describe("Transforms", () => {
+    beforeEach(() => {
+      cy.deleteAllIndices();
+
+      // Set welcome screen tracking to test_transform_target
+      localStorage.setItem("home:welcome:show", true);
+
+      // Go to sample data page_size
```

Comment:
Minor but why is it `page_size` here? Is it suppose to be just sample data page?